docs/guides/03-validation-proxy.md

```diff
@@ -0,0 +1,45 @@
+# Validation Proxy
+
+Unlike mocking, the validation proxy expects there to be a real API running somewhere. This could be a localhost environment, docker container, hosted sandbox environment, or deployed to actual production, doesn't matter.
+
+The proxy feature will help you identify discrepancies between the OpenAPI document (supposedly the source of truth) and any other server you designate as the proxy. This can help frontend developers integrating with your API, or other backend developers, who might want to channel their requests through Prism to see if they are making valid requests. The proxy can also be enabled in staging or any other pre-production environment, like a dress rehearsal before the opening night of a play.
```

Comment:
I'd say the server is not the proxy, it's the target

```suggestion
The proxy feature will help you identify discrepancies between the OpenAPI document (supposedly the source of truth) and any other server you designate as the target. This can help frontend developers integrating with your API, or other backend developers, who might want to channel their requests through Prism to see if they are making valid requests. The proxy can also be enabled in staging or any other pre-production environment, like a dress rehearsal before the opening night of a play.
```